doc/releases/quincy.rst

```diff
@@ -5,6 +5,1458 @@ Quincy
 Quincy is the 17th stable release of Ceph. It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+
+
+v17.2.8 Quincy
+==============
+
+This is the eighth backport release in the Quincy series. We recommend
```

Comment:
I don't think we are planning another quincy release -- should this be "... eighth, and expected to be last, backport ..."?